src/main/resources/application.yaml

```
@@ -50,6 +50,18 @@ upload:
   local:
     dir: ${UPLOAD_FILE_DIR}
 
+cloud:
+  aws:
+    credentials:
+      accessKey: ${AWS_ACCESS_KEY}
+      secretKey: ${AWS_SECRET_KEY}
+    s3:
+      bucket : ${AWS_BUCKET_NAME}
```

YAMLlint (1.37.1)

[warning] too many spaces before colon

(colons)